mobile/src/main/res/values/strings.xml

```diff
@@ -172,6 +170,12 @@
     <string name="intro_whatis">A vendor and technology agnostic open source automation software for your home</string>
     <string name="intro_themes">Themes</string>
     <string name="intro_themes_description">Choose between several themes</string>
+    <string name="try_again_button">Try again!</string>
```

Comment:
I'd prefer to omit the exclamation mark as we generally use a more formal style.